apps/allocations/app/components/Shared/LocalIdentityBadge.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react'
```

Comment:
If the code for this file is the same that the one used on Projects, I think it should be better to put it on shared/ui folder and import from there at both app, since we are going to put extra effort to de-duplicate these kind of files

Reply:
Yes. I'll update Projects along with modernizing the rest of the apps.